bluemira/equilibria/find_legs.py

```diff
@@ -234,34 +248,52 @@ def get_legs(
         return leg_dict
 
 
-def get_legs_length(
+def get_legs_length_and_angle(
     eq: Equilibrium,
     leg_dict: dict,
```

Comment:
I'd like slightly better typing here if possible. I think its this but please check
```suggestion
    leg_dict: dict[str, float | None],
```